package.json

```
@@ -31,10 +31,11 @@
     "cssnano": "4.1.10",
     "del": "6.0.0",
     "easymde": "2.10.2-360.0",
+    "gifsicle": "^5.1.0",
```

Version fixe pour ne pas avoir de soucis. Aussi je ne comprends pas pourquoi il y a besoin de la préciser explicitement car c'est une dépendance de dépendance de `gulp-imagemin` donc elle est déjà installée.
```suggestion
    "gifsicle": "5.1.0",
```